app.py

```diff
@@ -264,5 +266,58 @@ def get_pref(topic):
     )
 
 
+@app.route(f"{URL_PREFIX}/onboarding", methods=["GET", "POST"])
+@auth.requires_login
+def onboarding_survey():
```

Comment:
I might call this something like `demographic_survey` to leave room for future expansion of the things we ask on sign-up

Reply:
Alright, I changed that URL to `demographic_survey`.